roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
   - pip
+  - role: logrotate
+    logrotate_configs:
```

Comment:
Not positive how I feel about having the config values in meta. That being said, maybe that's just my lack of experience.